Cornelica/scenario/rp01a.txt

```diff
@@ -7,58 +7,60 @@
 ;;■CG01
 @bgf bg=ev01 t=1000
 
-　リリィがサキュバスらしい、淫靡な表情を浮かべたかと思うと、次の瞬間には押し倒されていた。
+;;リリィがサキュバスらしい、淫靡な表情を浮かべたかと思うと、次の瞬間には押し倒されていた。
+ Lily been succubus, she had a dubious look. I was pushed down the next moment.
 
-[リリィ/rry0013]「んふふ♪」
+[Lily/rry0013]"Fufu"
 
-　微笑むリリィの表情には淫靡な色が混ざっていた。
+;;微笑むリリィの表情には淫靡な色が混ざっていた。
+ The smiling Lily's expression had mixed lustrous colors
 
-[リリィ/rry0014]「どうしてこんなことをするのかって……？　こうやってHをして、魔力と一緒にチャームを吸い出すんだよ」
+[Lily/rry0014]"Why do this kind of thing? Doing it this way sucks the charm out with magic"
 
 @flash t=300 wt=0
 @bgv f=パイズリ・手コキ１ pitch=110
 
 @se f=愛液1 pitch=50
 
-[リリィ/rry0015]「おちんちん……凄く硬くなってるね。んふふ……ほら、どう？　こんな風にされるのは？　こうやって……おまんこで擦られるの……どう？」
+[Lily/rry0015]"Your penis... is getting really hard. Fufu, how is it? Why is it done like this? Doing this...rubbed in the vagina ... How is it?"
 
-[リリィ/rry0016]「ちょっと触れただけなのに凄く敏感な反応。全身が震えてる。おちんちんもビクビクッて飛び跳ねてるみたい」
+[Lily/rry0016]"I just touched bit but you got a very sensitive reaction. Your whole body is trembling. Your penis seems to be twitching too..."
 
 @se f=愛液1 pitch=50
 
-[リリィ/rry0017]「ふふ♪　目がトロンとしてる。気持ちよくて身体中から力が抜けちゃった？　頭が溶けちゃいそうなくらい感じちゃってるの？」
+[Lily/rry0017]"Fufu♪♪ Your eyes are torn. You feel comfortable and have gotten out of your body? Does it feel like your head is about to melt?"
 
 @se f=パイズリ・手コキ２ pitch=50
 
-[リリィ/rry0018]「でもね。もっとだよ。本番はここからなんだから……。ほら、んっ……んっ……んっ……どうかな？　こうやっておまんこのお肉でシコシコされるのは……」
+[Lily/rry0018]"However there is more... because the real thing start from here...uh...ah...uh...how is it? In this way the penis meat the vagina..."
```

Comment:
lol, `meat`

Reply:
Fuck, i had found the correct one later... It has like "your penis is tasting my vagina" or something like that...